content/_data/events/scale15x_2.adoc

```diff
@@ -0,0 +1,6 @@
+---
+title: "SCaLE 15x"
```

Comment:
Unless you change the title this will look pretty weird on the site, with three events having the same tile. Perhaps "SCaLE 15x Day 2"?